src/tlo/methods/diarrhoea.py

```diff
@@ -944,13 +944,14 @@ def do_at_generic_first_appt(
         self,
         patient_id: int,
         patient_details: PatientDetails,
+        symptoms: List[str],
         diagnosis_function: DiagnosisFunction,
         **kwargs,
     ) -> IndividualPropertyUpdates:
         # This routine is called when Diarrhoea is a symptom for a child
         # attending a Generic HSI Appointment. It checks for danger signs
         # and schedules HSI Events appropriately.
-        if patient_details.age_years > 5:
+        if (patient_details.age_years > 5) or "diarrhoea" not in symptoms:
```

Comment:
@marghe-molaro @tbhallett this is the only change I could see beyond fixing module ordering that might significantly affect model behaviour